frontend-react/src/components/StaticCompare.text.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
this filename should be `.test.tsx` otherwise jest won't pick it up

Reply:
Oh, typo!!! Great catch.